aligner/sbg-alignment-cwl/steps/pre-align.cwl

```diff
@@ -6,17 +6,29 @@ id: marko_zecevic/topmed-alignment/topmed-pre-align/1
 baseCommand:
   - samtools
   - view
-  - '-uh'
-  - '-F'
-  - '0x900'
 inputs:
   - id: input_file
     type: File
     inputBinding:
-      position: 0
+      position: 2
       shellQuote: false
-  - id: out_name
+    label: Input CRAM file
```

Comment:
The workflow definitely should accept a CRAM file so this is good, but I think it should also accept a BAM file. Will a BAM file input work as an input? Not sure if this is requirement because all of our TopMed open access files are CRAMs.

Reply:
I tried running it with BAM and it did work once I removed the secondary file requirement on that input (which we don't need anyway).